cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -24,6 +24,8 @@ method. It is not supported for non-default networks.
 * For secondary networks, use the `bridge` binding method.
 
 === Web console improvements
+* The {product-title} dashboard captures high-level information about clusters.
+From the OpenShift web console, access the dashboard by clicking *Home > Dashboards > Overview*. Note that virtual machines are no longer listed in the web console project overview. Virtual machines are now listed within the *Cluster Inventory* card of the dashboard.
```

Comment:
s/OpenShift/{product-title}
Each UI element must be [enclosed in its own set of asterisks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).